Scripts/Services/Monster Stealing/Core/StealingHandler.cs

```diff
@@ -166,7 +166,7 @@ private static int GetStealingChance(Mobile thief, BaseCreature from, double ste
 
             if (level >= 40)
                 chance += 5;
-            else if (level >= 40)
+             else if (level >= 40) /*??*/
```

Comment:
Assume 40, 35, 30.

Reply:
Will fix, i did browse uoguide and couldn't find anything related to fame for the formula so i was unsure of this one.